src/js/site.js

```diff
@@ -1,6 +1,15 @@
 (() => {
   const all = document.querySelectorAll.bind(document);
   const one = document.querySelector.bind(document);
+  const findUpByClass = (el, class_) => {
+    if (!el || !el.classList) {
+      return null;
+    }
+    if (el.classList.contains(class_)) {
+      return el;
+    }
+    return findUpByClass(el.parentNode, class_);
+  }
```

Comment:
I think this can be done with `closest` instead - checkout https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Reply:
I ALWAYS FORGET ABOUT CLOSEST, thanks for pointing this out!